examples/libfuzzer/libfuzzer_bin_example.cc

```diff
@@ -55,7 +55,11 @@ DEFINE_BINARY_PROTO_FUZZER(const libfuzzer_example::Msg& message) {
       !std::isnan(message.optional_float()) &&
       std::fabs(message.optional_float()) > 1000 &&
       message.any().UnpackTo(&file) && !file.name().empty()) {
+#if PROTOBUF_VERSION >= 4024000  // v3(!).24.0 (commit a6d895988dee9586b1689259f4412d1714ea1e53) for ShortFormat
     std::cerr << protobuf_mutator::protobuf::ShortFormat(message) << "\n";
+#else
+    std::cerr << message.DebugString() << "\n";
```

Comment:
We can switch back to `DebugString`.
All that StrCat/ShortFormat started as deprecation of DebugString.
I confirmed with libprotobuff authors that deprecation is abandoned.

We can use DebugString everywhere.

Reply:
@vitalybuka sounds good, I simplified in that regard, pushed